.changeset/slow-bugs-exist.md

```diff
@@ -0,0 +1,7 @@
+---
+'@clerk/clerk-js': patch
+'@clerk/shared': patch
+'@clerk/clerk-react': patch
+---
+
+Adds useAssertWrappedByClerkProvider to assert the presence of ClerkProvider
```

Comment:
````suggestion
Add `useAssertWrappedByClerkProvider` to internal code. If you use hooks like `useAuth` outside of the `<ClerkProvider />` context an error will be thrown. For example:
```shell
@clerk/clerk-react: useAuth must be used within the <ClerkProvider> component
```
````